profiler/src/main/java/com/splunk/opentelemetry/profiler/AgentInternalsFilter.java

```diff
@@ -16,29 +16,51 @@
 
 package com.splunk.opentelemetry.profiler;
 
-import java.util.Arrays;
-import java.util.function.Predicate;
+import java.util.stream.Stream;
 
-public class AgentInternalsFilter implements Predicate<String> {
+public class AgentInternalsFilter {
```

Comment:
It no longer just "agent internals" filter, this class is not responsible for general stack trace filtering. WDYT about `StackTraceFilter`?

Reply:
Sounds good.